### DISCUSSION TITLE
Drawio integration to custom Angular application

### DISCUSSION THREAD

**Discussion by Surinderk** [jgraph/drawio#3498] [Q&A]
We are looking to integrate Drawio to a custom application developed in Angular. We want to be able to drag and drop images from outside the drawio tool and be able to save/collect data about images outside the drawio. Do we have any documentation/support around this as we are finding it difficult to interact with drawio from outside the drawio?

**Comment by davidjgraph** [comment]
We don't have any documentation/support around this.
